### DISCUSSION TITLE
VIPS 8.13.3 Meson build and test succeeded but meson install fails

### DISCUSSION THREAD

**Discussion by tvngithub** [libvips/libvips#3904] [Help wanted]
Following the instruction to build VIPS on Ubuntu 22.04, I succeeded to build and test but the installation failed with
-------------------------------------
~/vips-8.13.3/build$ sudo meson install
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/mesonbuild/mesonmain.py", line 146, in run
    return options.run_func(options)
  File "/usr/lib/python3/dist-packages/mesonbuild/minstall.py", line 797, in run
    b = build.load(opts.wd)
  File "/usr/lib/python3/dist-packages/mesonbuild/build.py", line 2868, in load
    obj = pickle.load(f)
ModuleNotFoundError: No module named 'mesonbuild.utils'

ERROR: Unhandled python exception

    This is a Meson bug and should be reported!
----------------------------------------

First time for me to use Meson
Has anyone seen the same error and have a solution? It seems that the mesonbuild.utils is missing. 

Thank you very much.
-tom

**Comment by lovell** [comment]
It looks like you're using `sudo` and therefore require root-like access. How was meson installed? Make sure it's available system-wide, e.g. via `pip3 install meson`.

I also note you're not using the latest version of libvips, please upgrade to the latest if possible as many bugs will have been fixed since.

**Comment by tvngithub** [comment]
Thank you very much for your note. I found the fix for the problem and that was the difference between the mesa version installed via pip and the version that comes with Ubuntu 22.04. I removed mesa version that came with Ubuntu 22.04 and all is working.

What happened was the newest version installed with pip compiles and tests, but when I use "sudo" the version that came with Ubuntu 22.04 was evoked.
